test/test.ps1

```diff
@@ -96,7 +71,7 @@ $files | % {
     try {
         $framework ="netcoreapp${version}"
         exec {
-            (Get-Content (Join-Path $PSScriptRoot -ChildPath "Dockerfile.test.$image_os")).
+            (Get-Content (Join-Path $PSScriptRoot -ChildPath $docker_test_file)).
```

Comment:
One of the things I dislike about Powershell scripting is that variables have no scope.  I see the docker_test_file variable isn't passed in yet version, sdk_tag, and runtime_tag are but wouldn't strictly have to be.  Curious if there is a recommended best practice to follow in this situation.  At a minimum it seems like consistency should be used across all variables.